lib/swaps/Swaps.ts

```diff
@@ -12,32 +12,29 @@ import { StampedPeerOrder, StampedOwnOrder } from '../types/orders';
 
 type SwapDeal = {
   /** The role of the local node in the swap. */
+  // TODO: replace maker/taker with initiator/responder
   myRole: SwapDealRole;
-  /** The xud node pub key of the counterparty to this swap deal. */
-  peerPubKey: string;
   /** Global order id in the XU network. */
   orderId: string;
-  dealId: string;
   /** The quantity of the order to execute as proposed by the taker. Negative when the taker is selling. */
+  // TODO: is it needed here? if yes, should be in satoshis
```

Comment:
My thinking here was to have this quantity be in the same full units we use in the order book for easy subtraction/comparison. Proposed quantity is proposed by the taker, maker can respond with a lesser amount (which becomes the `quantity` field below.

Reply:
quantity and proposedquantity will be handled later. I don't see how this can/should be done without proper integration to the order book.

Reply:
Yes for sure, I didn't mean to suggest it has to be implemented in this PR, this fields are just placeholders. I was just explaining why I put it there and why I wasn't planning on using satoshis.

Reply:
Should this TODO be removed then?